Bottom: e66cc37e2617b7fd91400d9c10c8d3d335630c77
Top:    e396f327bf9ee25371268685b3fb6064d26f2a3e
Author: Jason Wang <jasowang@redhat.com>
Date:   2017-07-26 20:28:17 +0800

Refresh of tun-xdp-support

---

diff --git a/drivers/net/tun.c b/drivers/net/tun.c
index 82bbce4..9497a96 100644
--- a/drivers/net/tun.c
+++ b/drivers/net/tun.c
@@ -1506,8 +1506,6 @@ static ssize_t tun_get_user(struct tun_struct *tun, struct tun_file *tfile,
 		struct bpf_prog *xdp_prog;
 		int ret;
 
-		printk("generic!\n");
-
 		rcu_read_lock();
 		xdp_prog = rcu_dereference(tun->xdp_prog);
 		if (xdp_prog) {
